source/frontend/src/features/admin/financial-codes/add/__snapshots__/AddFinancialCodeForm.test.tsx.snap

```diff
@@ -375,17 +375,8 @@ exports[`AddFinancialCode form renders as expected 1`] = `
             class="react-datepicker-wrapper d-block"
           >
             <div
-              class="react-datepicker__input-container react-datepicker__view-calendar-icon"
+              class="react-datepicker__input-container"
```

Comment:
@FuriousLlama something funky caused all of these snapshots to change, is there a change to the FastDatePicker that I'm not seeing?

Reply:
This happened to me because I had not done a `npm install` after Devin updated the date-picker library in a recent PR. @FuriousLlama needs to do the same and the snapshots will align